platforms/m3/programming/mbus_snoop_snsv7_double_par_csv.py

```diff
@@ -4,12 +4,18 @@
 import sys
 import logging
 import csv
+import binascii
 
 import time
 import datetime
 from datetime import datetime
 
 
+#Adjust the default timezone for Cygwin
+from sys import platform
+if platform == 'cygwin':
+    os.system('export TZ="UTC+4"')
```

Comment:
Is this going to cause a problem when daylight savings time changes? (fuck time zones.. they're the worst.. :( )